tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -202,19 +222,21 @@ def branch(nodes: Union[TensorNetwork, Iterable[BaseNode]],
     nbranch: Number of best contractions to explore.
       If None it explores all inner products starting with those that
       have the best cost heuristic.
+    ignore_edge_order: An option to ignore the output edge order.
 
   Returns:
     The network after full contraction.
   """
   alg = functools.partial(
       opt_einsum.paths.branch, memory_limit=memory_limit, nbranch=nbranch)
-  return base(nodes, alg, output_edge_order)
+  return base(nodes, alg, output_edge_order, ignore_edge_order)
 
 
 def greedy(
     nodes: Union[TensorNetwork, Iterable[BaseNode]],
-    output_edge_order: Optional[Sequence[Edge]] = None,
-    memory_limit: Optional[int] = None):  # -> Union[BaseNode, TensorNetwork]:
+    output_edge_order: Sequence[Edge] = None,
```

Comment:
Don't remove `Optional` here.

Reply:
Sorry think this is just where my merge went wonky.